src/app/services/storage.service.ts

```diff
@@ -240,4 +240,16 @@ export class StorageService {
 
     return path.indexOf('/') < 0;
   }
+
+  // Takes a string with fuzzy units like 20Mib, 20k, 20Gb, returns number in bytes
+  convertHumanStringtoNum(str: string) {
+    if(!str) { return 0 }; // if empty, return the default 0, which is unlimited
+    const letters = str.replace( /[^a-zA-Z]/g, '');
+    if (letters[0] && !letters[0].toLowerCase().match(/b|k|m|g|t/)) { return NaN };
```

Comment:
This can be a character class like was used above.  The error handling probably needs to be more involved, though.

Reply:
...and that.